lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,104 @@
-    expect(moderate).toHaveProp('title', 'Moderate');
-    expect(severe).toHaveProp('title', 'Severe');
-    expect(critical).toHaveProp('title', 'Critical');
+    expect(threatIndicator).toHaveAttribute('aria-label', 'threat level low');
+    expect(threatIndicator).not.toHaveAttribute('aria-hidden', true);
   });
 
-  it('should show custom tooltip title', function() {
-    const tooltip = getShallowComponent({ title: 'Extinction Level Threat' }).find(NxToolip);
-
-    expect(tooltip).toHaveProp('title', 'Extinction Level Threat');
+  describe('should have default tooltips', function() {
+    allThreatLevelCategories.forEach(category => {
+      it(`for ${category} category`, async function() {
+        const { container } = quickRender({ threatLevelCategory: category }),
+            user = userEvent.setup(),
+            threatIndicator = container.querySelector('svg')!,
+            categoryTitleCase = category[0].toUpperCase() + category.slice(1);
+
+        await user.hover(threatIndicator);
+        const tooltip = await screen.findByRole('tooltip');
+        expect(tooltip).toHaveTextContent(categoryTitleCase);
+      });
+    });
   });
 
-  it('adds specified class names to the element', function() {
-    const icon = getShallowComponent({ className: 'foo', threatLevelCategory: 'low' }).find(NxFontAwesomeIcon);
+  it('should show custom tooltip title', async function() {
+    const { container } = quickRender({ title: 'Extinction Level Threat' }),
+        user = userEvent.setup(),
+        threatIndicator = container.querySelector('svg')!;
 
-    expect(icon).toHaveClassName('foo');
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toHaveClassName('nx-threat-indicator--low');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Extinction Level Threat');
   });
 
-  it('should hide tooltip when presentational prop is set true', function() {
-    const component = getShallowComponent({ presentational: true });
-    const tooltip = component.find(NxToolip);
-    const icon = component.find(NxFontAwesomeIcon);
+  it('should hide tooltip when presentational prop is set true', async function() {
+    const el = quickRender({ presentational: true }),
+        user = userEvent.setup(),
+        threatIndicator = el.getByRole('presentation', { hidden: true });
 
-    expect(tooltip).not.toExist();
-    expect(icon).toExist();
+    expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
+    await user.hover(threatIndicator);
+    expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
   });
 
   it('should set the role to "presentation" when presentational prop is true', function() {
-    const icon = getShallowComponent({ presentational: true }).find(NxFontAwesomeIcon);
+    const el = quickRender({ presentational: true }),
+        threatIndicator = el.getByRole('presentation', { hidden: true });
```

Comment:
I'm thinking the fact that they're hidden is a bug.  There's no point in us messing with the roles if they're always hidden, so presumably we didn't notice that they are when we implemented the `presentational` prop.  I suspect `aria-hidden={false}` should be added to the implementation.  That can be done in this ticket.

Reply:
> I'm thinking the fact that they're hidden is a bug. There's no point in us messing with the roles if they're always hidden, so presumably we didn't notice that they are when we implemented the presentational prop. I suspect aria-hidden={false} should be added to the implementation. That can be done in this ticket.

Actually I now see that it's only hidden when we set `role="presentation"` and not in the other cases.  That seems redundant but fine (and it appears it's implemented in fontawesome, not by us).  That's fine I guess.  The cleanest way to test that, I think, would be to simply assert that it does _not_ render an element with role `img` when the `presentational` prop is true.

Reply:
> > I'm thinking the fact that they're hidden is a bug. There's no point in us messing with the roles if they're always hidden, so presumably we didn't notice that they are when we implemented the presentational prop. I suspect aria-hidden={false} should be added to the implementation. That can be done in this ticket.
> 
> Actually I now see that it's only hidden when we set `role="presentation"` and not in the other cases. That seems redundant but fine (and it appears it's implemented in fontawesome, not by us). The cleanest way to test that, I think, would be to simply assert that it does _not_ render an element with role `img` when the `presentational` prop is true.

Added new test here: 0185e49193add4d15213a9056fef1fc6c7f6cd45